### PR TITLE
Use enry-java and siva-java

### PR DESCRIPTION
**Pull request by bzz**
Simplify things dramatically by avoiding external processes for siva unpack and language detection

 - [x] use enry-java
 - [x] use siva-java from https://github.com/src-d/siva-java/pull/3 
 - [x] add simplified language extractions by default

### CODE REVIEW

File: local-install-siva-java.sh
Comment (ajnavarro) [inline local-install-siva-java.sh:1]: Why do you need to do this for siva-java and enry-java? I think it should be done using https://jitpack.io

File: src/main/scala/tech/sourced/berserker/SparkDriver.scala
Comment (ajnavarro) [inline src/main/scala/tech/sourced/berserker/SparkDriver.scala:100]: you should create a case class for this.

File: src/main/scala/tech/sourced/berserker/git/JGitFileIterator.scala
Comment (ajnavarro) [inline src/main/scala/tech/sourced/berserker/git/JGitFileIterator.scala]: sivaFileName should include the extension

**Review by ajnavarro — CHANGES_REQUESTED** [review]

File: local-install-siva-java.sh
Comment (bzz) [reply local-install-siva-java.sh:1]: You are right: all, bblfsh scala client, enry-java and siva-java will be replaced by ones from jitpack.io eventually, in spirit of https://github.com/src-d/berserker/pull/33

But right now it's not possible for enry-java (due to final artifact not including native dependencies https://github.com/src-d/backlog/issues/973) and is not very useful for siva-java (until https://github.com/src-d/siva-java/issues/5 is fixed, as with current setup it's easier to test custom branches).

So, for now I would suggest to keep building dependencies from source here, not to block this PR, until both above issues are resolved and all jars are published on jitpack. 

Then, in a subsequent PR, source build of all dependencies together will be replaced to jitpack artifacts.
To make sure this valuable feedback not lost - this work is tracked under https://github.com/src-d/berserker/issues/30

File: src/main/scala/tech/sourced/berserker/SparkDriver.scala
Comment (bzz) [reply src/main/scala/tech/sourced/berserker/SparkDriver.scala:100]: Good point. We shall have a case class for this in spark-api.

For now, if you are do not have a strong opinion, I would prefer keeping it as a tuple, simply because API is not settled yet and it's is easier to modify.

But that's a good point, as soon as structure of API is settled and we understand all the dependencies - we will add a case class.

Pleas let me know if that argument does make sense to you or you think that it blocks merging!

File: src/main/scala/tech/sourced/berserker/git/JGitFileIterator.scala
Comment (bzz) [reply src/main/scala/tech/sourced/berserker/git/JGitFileIterator.scala]: Sorry for confusion, addressed in 2042cb3

**Comment by bzz** [comment]
All review feedback addressed. 
Please, let me know if you think there is anything else that shall be done before merging it.

**Comment by bzz** [comment]
Ready to be merged

**Comment by bzz** [comment]
Merging, if there is no further discussion

**Pull request merged**
